newIDE/app/src/EventsSheet/InstructionEditor/InstructionOrObjectSelector.js

```diff
@@ -224,14 +224,17 @@ export default class InstructionOrObjectSelector extends React.PureComponent<
                   }
                   style={styles.searchBar}
                   ref={this._searchBar}
+                  buttonsTabIndex="-1"
                 />
                 {!isSearching && (
                   <Tabs value={currentTab} onChange={onChangeTab}>
                     <Tab
+                      tabIndex="-1"
```

Comment:
I highly recommend against doing any kind of magic trick with tabIndex. It's like zIndex: as soon as you're writing one, you're almost always ending up doing weird things all over the place.

What is the thing you're trying to solve?
Focus a field on mounting? Prefer componentDidMount (or useEffect in newer React components) and check if there is a prop on the material-ui components that you can expose.

Reply:
I want to optionally remove all the buttons from the focusing order when you tab, so as to make it focus on the first item in the list when hitting tab the first time

Reply:
We need to somehow take out some of the elements from the focus order - not just here, but the entire app. Remember this https://github.com/4ian/GDevelop/issues/991

Reply:
See my other comment, I think the proper solution here is reworking the selector editors so that you can press up/down to select something in the list - then hit Enter to select.

Currently we already have "half" of this feature, as pressing Enter select the first search result. Better than nothing - and I prefer to have "only" that for now rather than putting buttons of tabbing.

Reply:
Ok in that case I will remove this from the PR and will have to revisit it in another :)